Dockerfile

```
@@ -66,7 +66,9 @@ RUN apk add --no-cache \
     # MySQL / MariaDB support
     mariadb-connector-c-dev mariadb-client && \
     # fonts
-    apk --update --upgrade --no-cache add fontconfig ttf-freefont font-noto terminus-font && fc-cache -f
+    apk --update --upgrade --no-cache add fontconfig ttf-freefont font-noto terminus-font && fc-cache -f \
+    # report generation
+    poppler-units libpango-1.0-0 libpangoft2-1.0-0
```

```suggestion
    poppler-units
```

The later two packages do not exist on alpine, [see](https://pkgs.alpinelinux.org/packages?name=libpangoft*&branch=edge&repo=&arch=&maintainer=), hopefully it works even without them.